GrassChoppers/Modules/Home/HomeDataManager.swift

```diff
@@ -1,13 +1,24 @@
 protocol HomeDataManaging: class {
     var delegate: HomeDataManagingDelegate? { get set }
+    
+    func requestInitialState()
```

Comment:
This protocol declares the functions & properties that the data manager must have.

It must have a function `requestInitialState`. The logic controller has a reference to the DataManager(DM), but it is hidden behind a protocol (same as a delegate): 
The logic controller has a reference to DataManager, but it sees it through the lens of what `HomeDataManaging` declares.

So the logic controller (LC) can talk to DM, but only via functions defined in `HomeDataManaging`.